src/Nakama/PreserveAttribute.cs

```diff
@@ -0,0 +1 @@
+internal class PreserveAttribute : System.Attribute {}
```

Comment:
nit: We should leave a doccomment note about why this is needed and refer to the Unity engine. It will help if other developers are curious about why it's needed.

Also file needs the LICENSE header doccomment.